azurelinuxagent/ga/exthandlers.py

```diff
@@ -524,10 +527,45 @@ def report_ext_handlers_status(self):
 
             self.report_status_error_state.reset()
 
+        self.write_ext_handlers_status_to_info_file(vm_status)
+
+    @staticmethod
+    def write_ext_handlers_status_to_info_file(vm_status):
```

Comment:
If there's a caller of this method other than the unittest code, I cannot find it in the blizzard of PEP8 fixes you've included in this PR. In general, I've learned (from getting yelled at) that it's really best not to mix PEP-8 fixes with *any* substantive change.

Please consider splitting this into two PRs: one with PEP-8 fixes *only*, and the other with the substantive changes you want to make.

Reply:
got it. I'll remember this for the next time. 

The call is done at https://github.com/Azure/WALinuxAgent/pull/1384/files#diff-f8c44add0bc67ec987eff75282ec2308R530 (last line of `report_ext_handlers_status()` method)